setup.py

```diff
@@ -143,10 +149,143 @@ def _header_bad(self, path):
+              'version %s.' % ('.'.join(str(v) for v in geos_version),
+                               '.'.join(str(v) for v in GEOS_MIN_VERSION)),
+              file=sys.stderr)
+        exit(1)
+
+    geos_includes = geos_includes.split()
+    geos_libraries = []
+    geos_library_dirs = []
+    for entry in geos_clibs.split():
+        if entry.startswith('-L'):
+            geos_library_dirs.append(entry[2:])
+        elif entry.startswith('-l'):
+            geos_libraries.append(entry[2:])
+    geos_extra_extension_args = {}
+
+# Proj4
+conda = os.getenv('CONDA_DEFAULT_ENV')
+if conda in sys.prefix:
+    # Conda does not provide pkg-config compatibility, but the search paths
+    # should be set up so that nothing extra is required. We'll still check
+    # the version, though.
+    proj = find_executable('proj')
+    if proj is None or conda not in proj:
+        print(
+            'Proj4 %s must be installed in Conda environment "%s".' % (
+                '.'.join(str(v) for v in PROJ_MIN_VERSION), conda),
+            file=sys.stderr)
+        exit(1)
+
+    try:
+        proj_version = subprocess.check_output([proj],
+                                               stderr=subprocess.STDOUT)
+        proj_version = proj_version.split()[1].split('.')
+        proj_version = tuple(int(v.strip(',')) for v in proj_version)
+    except (OSError, IndexError, ValueError, subprocess.CalledProcessError):
+        warnings.warn(
+            'Unable to determine Proj4 version. Ensure you have %s or later '
+            'installed, or installation may fail.' % (
+                '.'.join(str(v) for v in PROJ_MIN_VERSION), ))
+
+    proj_includes = []
+    proj_libraries = []
+    proj_library_dirs = []
+
+else:
+    try:
+        proj_version = subprocess.check_output(['pkg-config', '--modversion',
+                                                'proj'])
+        proj_version = tuple([int(v) for v in proj_version.split('.')])
+        proj_includes = subprocess.check_output(['pkg-config', '--cflags',
+                                                 'proj'])
+        proj_clibs = subprocess.check_output(['pkg-config', '--libs', 'proj'])
+    except (OSError, ValueError, subprocess.CalledProcessError):
+        warnings.warn(
+            'Unable to determine Proj4 version. Ensure you have %s or later '
+            'installed, or installation may fail.' % (
+                '.'.join(str(v) for v in PROJ_MIN_VERSION), ))
+
+        proj_includes = []
+        proj_libraries = []
+        proj_library_dirs = []
+    else:
+        if proj_version < PROJ_MIN_VERSION:
+            print(
+                'Proj4 version %s is installed, but cartopy requires at least '
+                'version %s.' % ('.'.join(str(v) for v in proj_version),
+                                 '.'.join(str(v) for v in PROJ_MIN_VERSION)),
+                file=sys.stderr)
+            exit(1)
+
+        proj_includes = proj_includes.split()
+        proj_libraries = []
```

Comment:
These defaults should take us back to what we had before. i.e. ``proj_libraries = ['proj']``